runtime/onert/backend/acl_common/IACLTensor.cc

```diff
@@ -60,6 +60,18 @@ int32_t IACLTensor::data_zero_point() const
   return info()->quantization_info().uniform().offset;
 }
 
+const std::vector<float> &IACLTensor::data_scales() const
+{
+  throw std::runtime_error("IACLTensor::data_scales() is not supported.");
+  return info()->quantization_info().scale();
```

Comment:
Please comment out return line. It may make SVACE warning - unreachable code.

Reply:
@hseok-oh I was worried that no return statement for non-void return function. But I will remove the return statements to eliminate unreachable code warning.